src/storage/statistics_cl.c

```diff
@@ -508,3 +518,255 @@ stats_dump (const char *class_name_p, FILE * file_p)
 
   fprintf (file_p, "\n\n");
 }
+
+/*
+ * stats_ndv_dump () - Dumps the NDV about a class
+ *   return:
+ *   classname(in): The name of class to be printed
+ *   fp(in):
+ */
+void
+stats_ndv_dump (const char *class_name_p, FILE * file_p)
+{
+  MOP class_mop;
+  int error = NO_ERROR;
+  DB_ATTRIBUTE *att;
+  int col_cnt = 0;
+  CLASS_ATTR_NDV class_attr_ndv;
+  int i;
+
+  class_mop = sm_find_class (class_name_p);
+  if (class_mop == NULL)
+    {
+      return;
+    }
+
+  /* get NDV by query */
+  if (stats_get_ndv_by_query (class_mop, &class_attr_ndv, file_p) != NO_ERROR)
+    {
+      goto end;
+    }
+
+  /* print NDV info */
+  fprintf (file_p, "\nNumber of Distinct Values\n");
+  fprintf (file_p, "****************\n");
+  fprintf (file_p, " Class name: %s\n", class_name_p);
```

Comment:
```suggestion
  fprintf (file_p, " Class name: %s\n", sm_get_ch_name (class_mop));
```

The table name entered by the user is displayed as it is.
Same table, but with different displayed names.

```
/* AS-IS */
csql> ;info ndv game        /* Class name: game */
csql> ;info ndv dba.game    /* Class name: dba.game */
csql> ;info ndv public.game /* Class name: public.game */
```

So, for consistent display, it is an opinion that class_name should be output using the sm_get_ch_name function.

```
/* TO-BE */
csql> ;info ndv game        /* Class name: public.game */
csql> ;info ndv dba.game    /* Class name: public.game */
csql> ;info ndv public.game /* Class name: public.game */
```